static/nextant.sh

```diff
@@ -95,11 +108,13 @@ cd $NC_APPS_PATH
 tar zxf $NT_RELEASE
 bash $SCRIPTS/setup_secure_permissions_nextcloud.sh
 rm -r $NT_RELEASE
-sudo -u www-data php $NCPATH/occ app:enable nextant 
+sudo -u www-data php $NCPATH/occ app:enable nextant
+
 if [ $? -eq 0 ]
 then
     echo "Nextant app is now installed and enabled."
     echo "Please go to: Admin Settings --> Additional Settings, and configure the app"
+    echo -e "Finally run \033[0;32msudo -u www-data php /var/www/nextant/occ nextant:index \033[0mto get nextant running."
```

Comment:
Not until the settings on the nextant app are saved on the admin settings.
Maybe there is a "cli" way of doing it, not sure about it. 
But before that it can't be done.